main.tf

```diff
@@ -42,6 +42,7 @@ resource "aws_iam_role" "role" {
             "ec2:DescribeAccountAttributes",
             "ec2:GetEbsEncryptionByDefault",
             "ec2:SearchTransitGatewayRoutes",
+            "ec2:GetTransitGatewayMulticastDomainAssociations",
```

Comment:
please keep them in lexicographical manner

Reply:
Ok

Reply:
Fixed.